BUILD.rst

```diff
@@ -55,13 +55,13 @@ CDAP Sandbox and Distributed CDAP
 - Build CDAP Sandbox distribution ZIP::
 
     MAVEN_OPTS="-Xmx1024m" mvn clean package \
-    -pl cdap-standalone,cdap-app-templates/cdap-etl,cdap-app-templates/cdap-data-quality,cdap-app-templates/cdap-program-report,cdap-examples \
+    -pl cdap-standalone,cdap-app-templates/cdap-etl,cdap-app-templates/cdap-data-quality,cdap-app-templates/cdap-program-report2_2.11,cdap-examples \
```

Comment:
we should have build similar to `cdap-etl` when you add spark1 PR. building program report should build both sub modules.